frontend/public/components/storage/create-pvc.tsx

```diff
@@ -80,14 +81,20 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     this.setState({ nameValuePairs }, this.onChange);
   };
 
+  handleCephProvisioner = (sc: string) => {
```

Comment:
`sc` is a bad variable name since it's not the storage class name, it's the provisioner name. `handleCephProvisioner` is also a confusing name since it's not always a Ceph provisioner.

```suggestion
  getStorageClassForProvisioner = (provisioner: string) => {
```

Reply:
ack